### ISSUE TITLE
Red slash

### ISSUE THREAD

**Issue by kingkai808** [pokesniperandroid/PokeSniper-Android#262]
Sniping with this WILL get you red slash.
Do not snipe 3rd evolution. Snipe 1st or 2nd and evolve to avoid slash. New secret update was launched last night

**Comment by kingkai808** [comment]
Wait for beta 21 update

**Comment by ebkeong** [comment]
Not true. I sniped a perfect IV Bulbasaur and evolved it to Ivysaur.  Also received a red crossout yesterday. 😫

**Comment by kingkai808** [comment]
Because you used this app. It has wrong api

**Comment by kymura30** [comment]
What is the deadline for BETA 21 to be released?

**Comment by ebkeong** [comment]
I see. Hope to see new update. This is the only app I use to snipe .

**Comment by kingkai808** [comment]
Hmm doesn't make sense. Sniped a 100 775 cp larvitar on 50 ACCTS, only 39 of them had red slash after evolved to pupitar. Out of those 10 were slashed as tyranitar. Of the ones not slashed they are played where it was physically caught at coords. Manually by me.

**Comment by kingkai808** [comment]
@pokesniperandroid 
Seen a beta 21 out but was in Spanish is that your work or copy cat? Available to download on a discord server.

**Comment by pokesniperandroid** [comment]
not ours, beta 21 released just now.

**Comment by amirmuntaha** [comment]
Thanks @pokesniperandroid
